Assignment_1/scikit.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+from sklearn.neighbors import KNeighborsClassifier
+np.set_printoptions(precision=2)
+ftrain = open("TrainingData_A1.tsv")
```

Comment:
when you want to open a file using open() method, it is better to close the file by close() method or you may want to use the `with` statement to close the file for you:)